src/renderer/browser/components/RequestsProcessor/Invoke/index.js

```diff
@@ -33,14 +33,14 @@ export default function makeInvoke(invokeActions, balancesActions) {
     // Ensure the arguments provided are valid
     withValidation(validateInvokeArgs),
 
-    // Prompt user
-    withInvocationPrompt(balancesActions),
-
     // Get the current network, account data, and priority fee data
     withNetworkData(),
     withData(authActions, mapAuthDataToProps),
     withData(feeActions, mapFeeDataToProps),
 
+    // Prompt user
+    withInvocationPrompt(balancesActions),
```

Comment:
It's moved to be able to access the fees from feesAction, right?

Reply:
Correct 😄